### PR TITLE
Pattern idea: duplicated-projects.md

### PR DESCRIPTION
**Pull request by spier**
Hi @rrrutledge. 

In Oct 2018 you prepared a Christmas gift for you, likely without knowing. Your gift can be seen in this PR.
It contains a potentially great Pattern idea, that is just waiting to be finished :)
I discovered this content in an un-merged and seemingly stale branch but it does seem to have some potential.

So what do you say to your own content from 2 years ago? :)

### CODE REVIEW

File: duplicated-projects.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Title
+Duplicated Projects
+
+## Patlet
+Forthcoming ...
+
+## Problem
+* Multiple teams are doing inner source, but they don't want to work together.
```
Comment (lenucksi) [inline duplicated-projects.md]:
```suggestion
* Multiple teams are doing InnerSource, but they don't want to work together.
```

File: duplicated-projects.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Multiple teams are working on the same product.  How do they merge?
+
+## Story (Optional)
+e.g. Splunk. 
+
+* 4 unique products have significant overlap.  Discovered after inner sourcing.
```
Comment (lenucksi) [inline duplicated-projects.md]:
```suggestion
* 4 unique products have significant overlap.  Discovered after opening them up as InnerSource projects.
```

File: duplicated-projects.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* This could apply to move than just inner sourcing situations
```
Comment (lenucksi) [inline duplicated-projects.md]:
```suggestion
* This could apply to move than just InnerSource situations
```

File: duplicated-projects.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## Context
+* Multiple teams (doing inner source or not) are trying to tackle the same problem.
```
Comment (lenucksi) [inline duplicated-projects.md]:
```suggestion
* Multiple teams (doing InnerSource or not) are trying to tackle the same problem.
```

**Review by lenucksi — COMMENTED** [review]
Nice, looking forward to see what becomes of this. Added a bunch of edits and whoever works on it probably wants to move this to `1-initial/`.

File: duplicated-projects.md (hunk 3 above)
Comment (spier) [reply duplicated-projects.md]:
```suggestion
* This could apply to more than just InnerSource situations
```

File: duplicated-projects.md (hunk 2 above)
Comment (spier) [reply duplicated-projects.md]: I worked this in via a separate commit. Thanks for the fix @lenucksi !

**Comment by spier** [comment]
I did further cleanup and moved the pattern to the `1-initial` folder.
Confirming with @rrrutledge now if he is happy to get this merged as is.

File: patterns/1-initial/duplicated-projects.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+* Multiple teams (doing InnerSource or not) are trying to tackle the same problem.
+* Managers want to run the project in their way.
+* Manager B gets upset if something Manager A does breaks B's team.
+* It's easier to be mean when you're not face-to-face.
+
+## Forces
+
+* People don't want to lose their jobs (or reason for existence).
+* Mangers can be territorial with their project.
```
Comment (spier) [inline patterns/1-initial/duplicated-projects.md]:
```suggestion
* Managers can be territorial with their project.
```

**Comment by spier** [comment]
Confirmed with @rrrutledge and will merge this pattern as a Donut.

**Pull request merged**